### ISSUE TITLE
Sharing - Clarify Not For All Users..

### ISSUE THREAD

**Issue by d6gNMGiKx6k8** [bitwarden/help#466]
EDIT HELP ARTICLE SUGGESTION:  https://bitwarden.com/help/article/share-to-a-collection/

TLDR: I suspect the downvotes are likely from people looking to share, but then they find out they can't share, and worse, they don't know why they can't share.  The word "personal" is confusing.  Family orgs should be clarified as examples, too.

Clarify the rule: Sharing is limited to org users (family, teams, enterprises) and is limited to only users within the org.  Emergency access is a separate thing (but still requires the emergency access user to have an account established ahead of time).

Additional Context:

While it is in the "organizations" category, and it says "collections are used by organizations to...", I noticed that the "rate this article" voting has twice the down votes.  I believe this is because it is not clear that ONLY organization users (family, or business plans) can share (not counting emergency access).  

The first sentence near the beginning says "All users can share items to an assigned Collection from their personal Vault..." and the first heading says "Share Existing Items From Your Personal Vault".  While technically accurate, many would read that heading, and that first sentence, and still be confused why they can't share.  They can't share because they aren't in an organization (family or a business) and sharing is limited to orgs.  Additionally, sharing is limited WITHIN the org.  I don't believe this is clear, either.

Much of the "organization" help topics assumes business users, but family "orgs" are likely increasing and should be clarified in the differences (even when minor).  And "personal" should be clarified when meaning a single individual's "personal" vault (regardless if the user is solo, family, team, enterprise), versus "personal" meaning "non-business" (non-team, non-enterprise and/or non-family?).  Again, to some, that first title and sentence appears to apply to non-business users, but it really means a single user.  "Personal" can be confusing. 

I believe there are three "org" types - family, team, and enterprises.  This is confusing when features differ.  For example, I still don't know if I can create user groups for my family - the https://bitwarden.com/help/article/what-plan-is-right-for-me/ page lists it as an item under teams and enterprises, but I know groups are tied to a collection (and I create collections in a family org).  Yet the https://bitwarden.com/help/article/about-groups/ page mentions it all as "business" users (marketing team, etc.).  If this doesn't apply to family orgs, then we should clarify that fact.  If it does apply to all orgs (family, team, enterprise) then examples should include husband, wife, kids, sister, etc.

TLDR (same as noted at top): I suspect the downvotes are likely from people looking to share, but then they find out they can't share, and worse, they don't know why they can't share.  The word "personal" is confusing.  Family orgs should be clarified as examples, too.

Clarify the rule: Sharing is limited to org users (family, teams, enterprises) and is limited to only users within the org.  Emergency access is a separate thing (but still requires the emergency access user to have an account established ahead of time).

Also, related pages should likely reference "emergency access" - as some people may be looking for that, without knowing it.

~ Hope this helps

**Comment by fschillingeriv** [comment]
@d6gNMGiKx6k8 this is some excellent perspective, thank you for taking the time to write up your thoughts! We'll get to work now on some edits to this particular article, but also in the future look for where these same points of confusion might also crop up.

**Comment by d6gNMGiKx6k8** [comment]
Just saw the "Create A Send" feature - that'll help a lot to include, as well!  That may be exactly what people want when finding this help page, so a very cool feature!
